### DISCUSSION TITLE
Tuning server assumptions

### DISCUSSION THREAD

**Discussion by dostrelith678** [le0pard/pgtune#55] [Q&A] [answered]
Hello, and thanks for the great tool for PG tuning!

I am interested in whether there are some assumptions made by the tuner to create its optimized settings (for example, the full hardware spec can be consumed by Postgres). Basically, if you could please answer a question like: "If only 50% of my server resources are available for Postgres, am I better off passing in 50% of my server resources to the tuner?"

**Comment by le0pard** [comment, marked as answer]
Hello @DamjanOstrelic . Sorry for delay. 

Math model try to calculate settings based on provided hardware. So, for example, if your server have 32 GB of RAM, but you can provide only 16 GB for PostgreSQL, because other 16 GB will be busy by other stuff, you need provide 16 GB for pgtune, because it is exactly what you trying to achieve. Calculations will provide settings, which will allow PostgreSQL in maximum load consume all provided RAM.
